hbase-migration-tools/bigtable-hbase-replication/bigtable-hbase-replication-core/src/main/java/com/google/cloud/bigtable/hbase/replication/metrics/HBaseToCloudBigtableReplicationMetrics.java

```diff
@@ -24,4 +24,9 @@ private HBaseToCloudBigtableReplicationMetrics() {}
   public static final String INCOMPATIBLE_MUTATION_METRIC_KEY = "bigtableIncompatibleMutations";
   public static final String DROPPED_INCOMPATIBLE_MUTATION_METRIC_KEY =
       "bigtableDroppedIncompatibleMutations";
+
+  public static final String INCOMPATIBLE_MUTATION_DELETES_METRICS_KEY =
+      "bigtableIncompatibleDeleteMutations";
+  public static final String INCOMPATIBLE_MUTATION_TIMESTAMP_OVERFLOW_METRIC_KEY =
+      "bigtableIncompatibleMutationsTimestampOverflow";
```

Comment:
Please be consistent in naming. It should either be bigtableIncompatibleTimestampOverflowMutation or other one should be bigtableIncompatibleMutationDelete

Reply:
I prefer bigtableIncompatibleTimestampOverflowMutation